packages/frontend/src/components/common/SpaceActionModal/SpaceActionModal.style.ts

```diff
@@ -0,0 +1,15 @@
+import { Colors } from '@blueprintjs/core';
+import styled from 'styled-components';
+
+export const RadioDescription = styled.p`
+    color: ${Colors.GRAY3};
+    margin: -5px 0px 10px 25px;
```

Comment:
can you use [Form group](https://blueprintjs.com/docs/#core/components/form-group) instead with `label` and `helptext` to wrap radio buttons. if it will not give the best result can you use the same styles for helptext:

```css
color: #5f6b7c;
font-size: 12px;
```